generate_qwant.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+# to run this script you need kosmitk and the openmaptiles-tools loaded
+
+set -e
+
+KOSMETIK="node /home/antoine/dev/bin/kosmtik/index.js"
+#KOSMETIK=kosmtik
+
+for tiles in 'base' 'poi' 'lite'; do
+    tileset="openmaptiles_$tiles.yaml"
+    generate-sql $tileset > generated_$tiles.sql
+    generate-imposm3 $tileset > generated_mapping_$tiles.yaml
+    TMP_SOURCE=generated_${tiles}_tm2source.yml
+    generate-tm2source $tileset  --host="localhost" --port=5432 --database="osm" --user="osm" --password="osm" > $TMP_SOURCE
+    $KOSMETIK export $TMP_SOURCE > generated_source_$tiles.xml
+    rm $TMP_SOURCE
+done
```

Comment:
I would be great to create a makefile entry for this instead of a standalone script.